packages/web-pkg/src/services/passwordPolicy/passwordPolicy.ts

```diff
@@ -91,7 +90,7 @@ export class PasswordPolicyService {
     }
     const rules = {} as Record<string, unknown>
 
-    if (this.useDefaultRules()) {
+    if (enforcePassword && !this.hasRules()) {
       rules.mustNotBeEmpty = {}
     }
```

Comment:
We might have password fields, that don't require a password, that is in conflict with the always added not empty rule